tests/zk.rs

```diff
@@ -0,0 +1,364 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+#![feature(once_cell)]
```

Comment:
Do not use `once_cell` yet. Unless they changed really recently, it's an nightly experimental feature that is still marked as unstable.

It's okay to use night-only feature, but they should be marked as stable.